spec/syntax.md

```diff
@@ -168,9 +168,10 @@ and consists of:
 
 1. an optional list of _declarations_, followed by
 2. a _complex body_
+3. optional trailing whitespace
```

Comment:
I think this is distracting and unnecessary and would leave it out.

If we go on to allow preceding whitespace, we'd handle it differently.

Reply:
```suggestion
```